packages/login/src/views/StepTwo/StepTwoForm.tsx

```diff
@@ -36,14 +36,18 @@ import {
 
 import { IVerifyCodeNumbers } from '@login/login/actions'
 import { Ii18nReduxFormFieldProps } from '@login/utils/fieldUtils'
-
 import { PrimaryButton } from '@opencrvs/components/lib/buttons/PrimaryButton'
 import { ceil } from 'lodash'
 import { messages } from '@login/i18n/messages/views/stepTwoForm'
 
 const StyledMobile2FA = styled(Mobile2FA)`
   transform: scale(0.8);
 `
+const StyledH2 = styled.h2`
+  ${({ theme }) => theme.fonts.h2};
+  /* stylelint-disable-next-line opencrvs/no-font-styles */
+  font-weight: 400;
```

Comment:
Please forgive.  I couldnt figure out why this h2 in the Login application is not rendering like the h2s in the client app.  Seems to force a bold weight somewhere and I ran out of time.

Reply:
Well the `h2` font defined here does have a font weight of 600. https://github.com/opencrvs/opencrvs-core/blob/53972be1befa72d0e0d24f337b69e28f6d52810c/packages/components/src/components/fonts.ts#L77

Reply:
Yes, but it doesnt seem to bold in the same way in the Login app.

Reply:
Just going to use a different style

Reply:
@Zangetsu101 this is the Login app without the hack:

![Screenshot 2022-06-17 at 12 16 42](https://user-images.githubusercontent.com/3169954/174288279-0a36556a-839b-4e60-be7e-59df2e2085db.png)

.. same header in client:

![Screenshot 2022-06-17 at 12 15 44](https://user-images.githubusercontent.com/3169954/174288327-63fda1bf-983b-4f5d-b799-de4cb8b0a0ed.png)